ConsoleApp2/Menu.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp2
+{
+    public class Menu
+    {
+        public static void PrintMenu(MenuOptions selectedOption)
+        {
+            Console.WriteLine("MENU:");
+
+            Console.WriteLine(selectedOption == MenuOptions.Calculator ? $"{InterfaceItems.Arrow} {MenuOptions.Calculator}" : $"    {MenuOptions.Calculator}");
```

Comment:
Your string always ends with {MenuOptions.Calculator}, so you can extract it to avoid duplicating code